redis/spring-cloud-starter-stream-sink-redis/src/main/java/org/springframework/cloud/stream/app/redis/sink/RedisSinkConfiguration.java

```diff
@@ -43,6 +45,7 @@
 @Configuration
 @EnableConfigurationProperties(RedisSinkProperties.class)
 @EnableBinding(Sink.class)
+@Import(SpelExpressionConverterConfiguration.class)
```

Comment:
yeah, I have created https://github.com/spring-cloud/spring-cloud-stream-app-starters/issues/167 to fix this in all the apps for GA.